mantis-runtime/src/main/java/io/mantisrx/runtime/KeyValueStageConfig.java

```diff
@@ -33,7 +33,12 @@ public abstract class KeyValueStageConfig<T, K, R> extends StageConfig<T, R> {
     private final Codec<K> keyCodec;
 
     public KeyValueStageConfig(String description, Codec<?> inputKeyCodec, Codec<T> inputCodec, Codec<K> outputKeyCodec, Codec<R> outputCodec, INPUT_STRATEGY inputStrategy, List<ParameterDefinition<?>> params) {
-        super(description, inputKeyCodec, inputCodec, outputCodec, inputStrategy, params);
+        super(description, inputKeyCodec, inputCodec, outputCodec, inputStrategy, params, DEFAULT_STAGE_CONCURRENCY);
```

Comment:
nit: you could call the other constructor in this class instead of duplicate call to super + keyCodec assignment.